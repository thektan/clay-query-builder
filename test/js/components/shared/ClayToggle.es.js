import React from 'react';
import ClayToggle from 'components/shared/ClayToggle.es';
import {cleanup, render} from 'react-testing-library';

describe(
	'ClayToggle',
	() => {
		afterEach(cleanup);

		it(
			'should render',
			() => {
				const {asFragment} = render(
					<ClayToggle />
				);

				expect(asFragment()).toMatchSnapshot();
			}
		);
	}
);