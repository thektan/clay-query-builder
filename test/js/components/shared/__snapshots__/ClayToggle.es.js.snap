// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClayToggle should render 1`] = `
<DocumentFragment>
  <label
    class="toggle-switch"
  >
    <input
      class="toggle-switch-check"
      type="checkbox"
    />
    <span
      aria-hidden="true"
      class="toggle-switch-bar"
    >
      <span
        class="toggle-switch-handle"
        data-label-off=""
        data-label-on=""
      />
    </span>
  </label>
</DocumentFragment>
`;
